distributions/nuxt-start/package.json

```diff
@@ -55,8 +55,12 @@
     "dist"
   ],
   "dependencies": {
+    "@nuxt/builder": "2.12.1",
```

Comment:
`nuxt-start` is a special package for using in production without webpack dependency. I think we should not list it here as require won't even happen for it :)